src/components/admin/SpaceEditForm.tsx

```diff
@@ -49,8 +49,10 @@ import { useRelatedVenues } from "hooks/useRelatedVenues";
 import { BackgroundSelect } from "pages/Admin/BackgroundSelect";
 
 import { AdminSidebarButtons } from "components/organisms/AdminVenueView/components/AdminSidebarButtons";
+import { RunTabUsers } from "components/organisms/AdminVenueView/components/RunTabUsers/RunTabUsers";
```

Comment:
This component could use a rename and move to a simpler path. Maybe something boring like
```suggestion
import { SpaceUsersManager } from "components/admin/SpaceUsersManager";
```

Reply:
[dd0b65e](https://github.com/sparkletown/sparkle/pull/2766/commits/dd0b65efe95419fd2b89c4f195ef5269cb7bd6e9)